includes/class-wc-stripe-update-manager.php

```
@@ -0,0 +1,44 @@
+		( new Migrate_Payment_Request_Data_To_Express_Checkout_Data() )->maybe_migrate();
+
+		require_once WC_STRIPE_PLUGIN_PATH . '/includes/migrations/class-sepa-tokens-for-other-methods-settings-update.php';
+		( new Sepa_Tokens_For_Other_Methods_Settings_Update() )->maybe_migrate();
+
+		( new WC_Stripe_Express_Checkout_Add_Change_Payment_Method_Location_Update() )->maybe_migrate();
+
+		/**
+		 * Action triggered when the plugin is updated.
+		 */
+		do_action( 'woocommerce_stripe_updated' );
+	}
```

PHPMD (2.15.0)

[error] The method run_update_checks is not named in camelCase. (undefined)

(CamelCaseMethodName)